cloud/gcp/variables.tf

```diff
@@ -0,0 +1,16 @@
+variable "terraform_service_account" {
+  type        = string
+  description = "Service Account to use for running terroform on GCP."
+}
+
+variable "region" {
+  type        = string
+  default     = "us-west1"
+  description = "Default region for the project"
+}
+
+variable "company-name" {
```

Comment:
suggestion (blocking): same comment as above, but also does this need to be declared more than once?

Reply:
The idea behind modularization of TF code is that each module is a black box on its own. You can view 'storage' module and know all inputs used and transformed outputs. Declaring it as variables in each modules tells the user what needs to be supplied as if the module wasn't related to the 'main' code itself. Hence multiple declaration. Say a month from now someone wants to use the storage module here in a code in some other repo, they can do it easily by giving a github link as source and then mapping their variables to mine.

Also, makes for a more sophisticated directory structure :)

Reply:
Gotcha, thank you!